README.md

```diff
@@ -344,6 +368,8 @@ Note that if you already have a running cluster you could use upgrade command to
 $ helm upgrade -f values/values.dynamic_config.yaml temporaltest . --timeout 900s
 ```
 
+Note that doing so will trigger a rolling upgrade of your system.
```

Comment:
i'm not super familiar with helm, but would you want to add the `-f values/values.dynamic_config.yaml` /in addition/ to everything that was specified when initially deployed? otherwise wouldn't it use the defaults and change all sorts of things?

Reply:
Do you mean in all other sections, outside of "Updating Dynamic Config"?

I think the initial statement in this section and point # 3 (which has an example of deploying with dynamic configs) is probably sufficient – it keeps dynamic config complexity out of  other sections, while providing sufficient information to those who are specifically interested in dynamic configs.

What do you think?

Reply:
(I will go ahead and merge for velocity, but please let me know if you have more thoughts / feedback on this).

Reply:
sgtm